.circleci/config.yml

```diff
@@ -59,7 +59,10 @@ jobs:
       - test
 
   publish:
-    executor: node-10
+    executor:
+      name: node/default
+      tag: '10.20.1'
```

Comment:
Also curious what this is referring to.

Reply:
circle has image tags for each node version. Here's the complete list of pre-built docker images and tags - https://circleci.com/docs/2.0/docker-image-tags.json. With `name` and `tag`, we are telling circle to use the docker image that has the node version we want to use.